easypost/__init__.py

```diff
@@ -961,11 +961,10 @@ class Report(AllResource, CreateResource):
     def create(cls, api_key=None, **params):
         requestor = Requestor(api_key)
         url = "%s/%s" % (cls.class_url(), params['type'])
-        wrapped_params = {cls.class_name(): params}
-
-        response, api_key = requestor.request('post', url, wrapped_params, False)
+        response, api_key = requestor.request('post', url, params)
         return convert_to_easypost_object(response, api_key)
 
+
```

Comment:
methods within a class should only have 1 newline in between, per PEP8 (https://www.python.org/dev/peps/pep-0008/)

Reply:
I too thought this was strange but tried to keep it similar to what was already there. I've removed the extra lines from the Report class.